client_config_backend: settings
client_config:
  client_id: null
  client_secret: null

get_refresh_token: True
oauth_scope:
  - https://www.googleapis.com/auth/drive